src/__snapshots__/openApi.test.ts.snap

```diff
@@ -103,6 +103,52 @@ Object {
                             },
                             "type": "array",
                           },
+                          "eatenBy.userId": Object {
```

Comment:
It looks like these came out as objects instead of an array?

Reply:
Though if it generates correctly in the SDK, I'm ok with it.

Reply:
It'll be an object but have an `items` list.

Reply:
<img width="306" alt="Screenshot 2023-08-22 at 10 06 04 PM" src="https://github.com/FlourishHealth/ferns-api/assets/6036168/a47ff7b2-9d30-4aeb-9119-176b39acd0e3">
for example, this is what is generated for notifications.